business-central-parent/business-monitoring-webapp/pom.xml

```diff
@@ -1678,12 +1678,25 @@
         <artifactId>spotbugs-maven-plugin</artifactId>
       </plugin>
       <plugin>
-        <groupId>io.thorntail</groupId>
-        <artifactId>thorntail-maven-plugin</artifactId>
+        <groupId>org.wildfly.plugins</groupId>
+        <artifactId>wildfly-jar-maven-plugin</artifactId>
         <configuration>
-          <filterWebInfLib>uberjar-only</filterWebInfLib>
+          <cli-sessions>
+            <cli-session>
+              <script-files>
+                <script>../scripts/script.cli</script>
+              </script-files>
+            </cli-session>
+          </cli-sessions>
         </configuration>
       </plugin>
+<!--      <plugin>-->
+<!--        <groupId>io.thorntail</groupId>-->
+<!--        <artifactId>thorntail-maven-plugin</artifactId>-->
+<!--        <configuration>-->
+<!--          <filterWebInfLib>uberjar-only</filterWebInfLib>-->
+<!--        </configuration>-->
+<!--      </plugin>-->
```

Comment:
please remove commented old plugin

Reply:
Sure I will. This is mainly for Tomas to try this out, since we need to hold the merge before Wildfly 23 update and we might also need to update the bootable jar plugin version number.
This should provide a good test jar, since it will not contain any legacy security parts. Meaning it is impossible for the BC to use them. If this works the rest of the Elytron update should be possible to do with standalone.xml configuration changes and migration of the KieLoginModule class.